docker-compose.yml

```diff
@@ -77,6 +77,7 @@ services:
       DJANGO_LOG_SERVER_HOST: vector
       DJANGO_LOG_SERVER_PORT: 80
       no_proxy: clickhouse,grafana,vector,nuclio,opa,${no_proxy:-}
+      NUMPROCS: 1
```

Comment:
Probably, this change is from a local debug configuration and should not be committed?

Reply:
No, we have such environment variables for other containers and I guess that passing this env into utils container is also useful (as you can see it is used now for configuring the number of processes for rqworker)